### PR TITLE
ZEP-2766: NZ Contact Phone

### PR DESCRIPTION
**Pull request by zachingle**
- Update phones for contacts to be valid NZ mobile numbers
- Specify what format they must be in, and provide example of alternate format ([E.164](https://en.wikipedia.org/wiki/E.164) or [national](https://en.wikipedia.org/wiki/Telephone_numbers_in_New_Zealand#Mobile_phones))
- Semi-related: Update account numbers to use a valid BNZ branch code

Screen Shot 2022-11-11 at 11 32 32 am

### CODE REVIEW

**Review by aathithya-zepto — APPROVED** [review]

File: source/openapi3/split.yaml [hunk 1]
```diff
@@ -3209,7 +3209,7 @@ components:
           description: The email of the Contact (256 max. characters)
         phone:
           type: string
-          description: The phone number of the Contact (11 max. characters)
+          description: The phone of the Contact. Must be a valid New Zealand mobile number in [ITU-T E.164](https://en.wikipedia.org/wiki/E.123) or [national](https://en.wikipedia.org/wiki/Telephone_numbers_in_New_Zealand#Mobile_phones) format
```
Comment (kieranjogrady) [inline source/openapi3/split.yaml]: Would it be more relevant to link to the E.164 page?
```suggestion
          description: The phone of the Contact. Must be a valid New Zealand mobile number in [ITU-T E.164](https://en.wikipedia.org/wiki/E.164) or [national](https://en.wikipedia.org/wiki/Telephone_numbers_in_New_Zealand#Mobile_phones) format
```

Comment (zachingle) [reply source/openapi3/split.yaml]: Woops, it definitely would be 😅

**Review by kieranjogrady — APPROVED** [review]

**Review by henripryde400 — APPROVED** [review]
Looks good to me. Just confirming that the number in the sample payload will pass in Sandbox?

**Comment by zachingle** [comment]
> Looks good to me. Just confirming that the number in the sample payload will pass in Sandbox?

Yep! Replaced all contact numbers with valid NZ mobile numbers

**Pull request merged**